ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/utils/useJoplinMode.ts

```diff
@@ -0,0 +1,25 @@
+import 'codemirror/addon/mode/multiplex';
+import 'codemirror/mode/stex/stex';
+
+// Joplin markdown is a the same as markdown mode, but it has configured defaults
+// and support for katex math blocks
+export default function useJoplinMode(CodeMirror: any) {
+	CodeMirror.defineMode('joplin-markdown', (config: any) => {
+		const stex = CodeMirror.getMode({ inMathMode: true }, 'stex');
```

Comment:
It looks like you use a mode specific for math expressions, so I'm wondering can we get this highlighted as well? On [this page](https://codemirror.net/mode/stex/), they have the highlighting but I'm not sure if it can be done when it's within Markdown. It would be great if it was though!

Reply:
I'll take another stab at this. It should be as easy as including `parseDelimiter: true` but that doesn't seem to work.

Reply:
> It should be as easy as including parseDelimiter: true

Yes that's what I was wondering. If it's as easy as switching a flag on, then let's do it but if it's more complicated don't worry about it, just let me know either way.

Reply:
It turned out to be as easy as switching a flag *correctly*